dev_guide/application_lifecycle.adoc

```diff
@@ -12,46 +12,56 @@ toc::[]
+As a PaaS, OpenShift is built for building and deploying applications. Depending
+on how much you want to involve OpenShift in the development process, you can
+choose to develop on OpenShift after it has been installed, and use it to
+continuously develop an application, or you can deploy a fully developed
+application onto a newly installed OpenShift instance.
 
 == Developing on OpenShift
 
-If you plan to use develop your application on OpenShift directly:
+You can develop your application on OpenShift directly. Use the following if you
+plan to use OpenShift as a method to build and deploy your application:
 
-Plan::
+Initial Planning::
 * What does your application do?
-* What programming language will you be developing in?
+* What programming language will it be developed in?
  
 Install::
 * OpenShift should be installed by this point, either by yourself or an
 administrator within your organization.
 
-Develop::
-* This is where you develop your application code.
-
 Generate::
-* link:../dev_guide/new_app.html[Create an basic application] so that OpenShift can determine the programming language you are using. OpenShift generates build and deployment configurations.
+* link:../dev_guide/new_app.html[Create a basic application], esseitially creating
+an application framework, so that OpenShift can determine the programming
+language you are using. Here, OpenShift generates build and deployment
+configurations.
+* Start developing your application code.
 
-Code/Manage/Verify::
+Code/Manage::
 * How will you continue to develop your application? 
-* Remember to periodically verify that your application code is running
-successfully.
+* Continue to develop and polish your code, pushing your changes to your
+application's GitHub repository.
+
+Verify::
+* Depending on your preferences, you have a number of ways to verify your
+application. You can push your changes to your application's GitHub repository,
+and use OpenShift to rebuild and redeploy your application. Alternatively, you
+can hot deploy using rsync to synchronize your code changes into a running pod.
 
 == Developing Then Deploying on OpenShift
 
 Another possible application life cycle is to develop locally, then use
-OpenShift to deploy a fully developed application:
+OpenShift to deploy your fully developed application. Essentially, use the
+following if you have plan to have application code already, then want to build
+and deploy onto an OpenShift isntallation after:
 
-Plan::
+Initial Planning::
 * What does your application do?
-* What programming language will you be developing in?
+* What programming language will it be developed in?
 
 Develop::
-* This is where you develop your application code. The steps can vary depending on
+* Develop your application code. The process can vary depending on
 how you will be installing OpenShift. Will you install a local instance? Or are
 you installing within an existing OpenShift infrastructure in your organization?
```

Comment:
the process of actually developing the code doesn't depend on openshift, since this is the "develop outside of openshift" flow.